includes/class-wc-payments-features.php

```diff
@@ -276,6 +276,23 @@ public static function is_fraud_protection_welcome_tour_dismissed(): bool {
 		return '1' === get_option( 'wcpay_fraud_protection_welcome_tour_dismissed', '0' );
 	}
 
+	/**
+	 * Checks whether Simplify Deposits UI is enabled. Enabled by default.
+	 *
+	 * @return bool
+	 */
+	public static function is_simplify_deposits_ui_enabled(): bool {
+		return '1' === get_option( self::SIMPLIFY_DEPOSITS_UI_FLAG_NAME, '1' );
+	}
```

Comment:
There is so merge conflict issue that's being resolved. Please ignore this block